api-gateway/src/main/java/com/petclinic/bffapigateway/presentationlayer/BFFApiGatewayController.java

```diff
@@ -326,8 +326,8 @@ public Mono<Role> addRole(@RequestHeader(AUTHORIZATION) String auth, @RequestBod
      * **/
 
     @GetMapping(value = "owners")
```

Comment:
should produce TEXT_EVENT_STREAM_VALUE

Reply:
What do you mean ?

Reply:
So this should be just fine 
``` $http.get('api/gateway/owners').then(function (resp) {
            self.owners = resp.data;
             console.log(resp)
        }); ```

Reply:
> TEXT_EVENT_STREAM_VALUE

you mean this ``@GetMapping(value = "owners"}, produces = MediaType.TEXT_EVENT_STREAM_VALUE)``

Reply:
> We are using AngularJS which is a framework that is not inherently reactive like Angular (with RxJS) or other modern frontend frameworks. So basically AngularJS doesn't have built-in support for reactive programming.

It actually is possible to do, I sent you a code snippet and posted it in the general chat on Teams.